frontend/src/utils/Pane.svelte

```diff
@@ -10,117 +10,31 @@
     flex-flow: row;
     justify-content: space-between;
     align-items: stretch;
+    overflow: hidden;
   }
 
   .inner {
-    overflow: auto;
+    overflow: hidden;
```

Comment:
This may have fixed some short-term problems, but I am really skeptical that this is not the cause of a bunch of other scroll things breaking.

I don't think this should have overflow hidden.